src/index.html

```diff
@@ -8,9 +8,9 @@
 	<meta name="description" content="The home of dojo. Documentation. Examples. Community.">
 	<link rel="icon" href="./assets/favicon.ico" />
 	<link rel="stylesheet" type="text/css" href="./assets/prism.css" />
-	<link rel="stylesheet" type="text/css" href="./assets/fontawesome.css" />
 </head>
 <body>
 	<div id="root"></div>
+	<link rel="stylesheet" type="text/css" href="./assets/fontawesome.css" />
```

Comment:
This is resolving a CSS ordering issue with the BTR build.

**Details**:
1. Load the home page first: https://site-kzxndlylhg.now.sh/home
2. The class `fa-4x` is not in the BTR css as it is not used.
3. Go to the tutorials or reference guides pages.
4. The icons are small. The `svg-inline--fa` class (from BTR index) has higher ordering than `fa-4x` (from post load CSS), resulting in the icons being small.
5. Load https://site-kzxndlylhg.now.sh/tutorials directly
6. The icons are correctly sized The `fa-4x` class (from BTR index) has higher specificity than `svg-inline--fa` (from BTR index), resulting in the icons being correctly sized.